Resources/views/frontend/_public/js/jquery.emotion.js

```diff
@@ -0,0 +1,76 @@
+$.overridePlugin('swEmotionLoader', {
```

Comment:
Dangerous.
You would have to keep track of every change from shopware itself in order to not re-include bugs.

Just as an example, [this](https://github.com/shopware/shopware/pull/1605/files) was **almost** merged into shopware and would have been reopened with your changes.